input/data/crop/example_alf_corn_silage_rotation.json

```diff
@@ -0,0 +1,105 @@
+{"crop_schedules":
+  [
+ 
+  {
+    "crop_species": "corn_silage",
+    "harvest_days": [
+      263
+    ],
+    "harvest_years": [
+      2013
+    ],
+    "harvest_operations": [
+      "harvest_kill"
+    ],
+    "harvest_type": "scheduled",
+    "planting_days": [
+      127
+    ],
+    "planting_years": [
+      2013
```

Comment:
Is it intentional to have this crop planted and harvested outside the example config simulation timeframe (2014-2019)?